resources/apt-packages.txt

```diff
@@ -0,0 +1,38 @@
+# utilities
+git
+mercurial
+pkg-config
+wget
+# debian-provided interpreters
+python-pip
+python2.7
+python2.7-dev
```

Comment:
Which packages in particular would you like to remove? I only added `libbz2-dev`, `libgdbm-dev`, `liblzma-dev`, `libncurses5-dev` and `libreadline-dev`. Most of the non-dev versions of these packages are installed already. The remaining packages are already there to support building user-specified libraries with python extensions.

We explicitly document which ones we support : https://cloud.google.com/appengine/docs/flexible/python/runtime#using_c_libraries_with_python

(Also, there's an idea I have of using the cloud build infrastructure to prepare wheels and then export them to a runtime that has no -dev packages, but that'll have to come later).